app/controllers/notes_controller.rb

```diff
@@ -0,0 +1,26 @@
+class NotesController < ApplicationController
+
+  def index
+    @notes = Note.all
+    render json: @notes
+  end
+
+  def create
+    @note = Note.new(note_params)
+    if @note.save
+      params[:tags].split(",").map(&:strip).each do |tag|
```

Comment:
After the note saves, we take params[:tags] (which, you'll notice _isn't_ in our strong params permits), split them on the commas, strip the whitespace, each over the resulting array and then tag the note.

Aside: Another approach to this would be to override `tags=` in our model to do much the same - and then we could just let params[:tags] go into Note.new. The downside to that approach in this case is that you're overriding a method that comes in handy in other ways.